packages/eliom/eliom.6.3.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "eliom"
+maintainer: "dev@ocsigen.org"
+authors: "dev@ocsigen.org"
+homepage: "http://ocsigen.org/eliom/"
+bug-reports: "https://github.com/ocsigen/eliom/issues/"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "https://github.com/ocsigen/eliom.git"
+build: [make]
+depends: [
+  "ocamlfind"
+  "deriving" {>= "0.6"}
+  "ppx_deriving"
+  "ppx_tools" {>= "0.99.3"}
+  "js_of_ocaml" {>= "3.0"}
+  "js_of_ocaml-lwt"
+  "js_of_ocaml-ocamlbuild" {build}
+  "js_of_ocaml-ppx"
+  ("js_of_ocaml-ppx" {<= "3.0.2"} | "js_of_ocaml-ppx_deriving_json")
```

Comment:
Is this line correct regarding the line above ?

Reply:
You need `js_of_ocaml-ppx` in any case (line 18). If you have `<= 3.0.2` `js_of_ocaml-ppx_deriving_json` was included in the `-ppx` package, so you don't need anything extra. In `3.1.0` you need to install the split-off package, hence the conditional.

Reply:
I see. Thanks !